### PR TITLE
propose reorganisation of installation doc

### PR DESCRIPTION
**Pull request by jphuart**
The file concerning install additional bundle is just as a draft, in fact I have to review it completely taking into consideration the icpc2 bundle. Can you confirm me that the bundles installed by default are main, person and report? What about custom-fields and icpc2?

All what concerns install should now enable a full functional test server. But I will retest it soon.

### CODE REVIEW

**Comment by jphuart** [comment]
Install additional bundle has been reviewed and will be updated once last aspect will be automated.

File: source/installation/install_additional_bundles.rst [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+
+   - :ref:`main-bundle`
+   - :ref:`person-bundle`
+   - :ref:`report-bundle`
+   - :ref:`custom-fields-bundle`
+   
+but you can add as many as needed by your project, and if the bundle does not exists yet, you can create a new one, see :ref:`create-new-bundle` .
+   
+In Chill you are free to do what is most suitable for your activity, so let's go into details on how to add an existing bundle.
+We will add the bundle 'icpc2' that does something very important...
```
Comment (julienfastre) [inline source/installation/install_additional_bundles.rst:24]: Proposal : 
We will add the bundle 'icpc2' that set `icpc code <https://en.wikipedia.org/wiki/International_Classification_of_Primary_Care>`_ available as custom field.

Comment (julienfastre) [inline source/installation/install_additional_bundles.rst:24]: I do not have access to review your branch and I do not know how to make changes before your pull request...

Comment (julienfastre) [inline source/installation/install_additional_bundles.rst:24]: done locally :-)

**Comment by julienfastre** [comment]
The pull request is merged !

Thanks for your ideas, @jphuart !

**Pull request merged**